src/Hint/List.hs

```diff
@@ -63,7 +63,7 @@ import GHC.Builtin.Types
 import GHC.Util
 import Language.Haskell.GhclibParserEx.GHC.Hs.Pat
 import Language.Haskell.GhclibParserEx.GHC.Hs.Expr
-import Language.Haskell.GhclibParserEx.GHC.Hs.Types
+import Language.Haskell.GhclibParserEx.GHC.Hs.Type
```

Comment:
This change is just the inverse of #1352 - is that deliberate?

Reply:
change was deliberate. i can't tell you why #1352 was made. it's not right and this fixes it.

Reply:
ah wait. i guess i get it. i failed to pull in this change. let me rebase and push again.

Reply:
right, so it wasn't the inverse, it was applying the same change because i failed to branch from head. have rebased and pushed again. that will sort this